test/JDBC/expected/babel_time-vu-verify.out

```diff
@@ -0,0 +1,925 @@
+-- psql
+-- behaviour of casting to time in psql
+SELECT CAST('2012-02-23' AS TIME)
+GO
+~~ERROR (Code: 0)~~
+
+~~ERROR (Message: ERROR: invalid input syntax for type time: "2012-02-23"
+  Position: 53
+    Server SQLState: 22007)~~
+
+
+-- tsql
+-- if only date is given as input
+-- accepted formats are `mm{-/.}dd{-/.}yyyy`, `yyyy{-/.}mm{-/.}dd`
+-- If text month is given then acceptable formats are `dd{-/.}Mon{-/.}yyyy`, `yyyy{-/.}Mon{-/.}dd`
+SELECT CAST('   23   -     11   -   2000   ' AS TIME)
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Conversion failed when converting date and/or time from character string.)~~
+
+
+SELECT CAST('23-2000-11' AS TIME)
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Conversion failed when converting date and/or time from character string.)~~
+
+
+SELECT CAST('11-2000-23' AS TIME)
```

Comment:
let us use different dates rather than same value. Also, include corner cases like 1 Jan, feb and dec etc.

Reply:
Sure, will add edge cases for different types of inputs.